src/lib/Mutation/AuthenticationMutation.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\GraphQL\Mutation;
+
+use Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Core\MVC\Symfony\Security\User;
+use Lexik\Bundle\JWTAuthenticationBundle\Services\JWTTokenManagerInterface;
+use Overblog\GraphQLBundle\Definition\Argument;
+
+final readonly class AuthenticationMutation
+{
+    public function __construct(
+        private JWTTokenManagerInterface $tokenManager,
+        private UserService $userService
+    ) {
+    }
+
+    /**
+     * @return array<string, ?string>
+     *
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     */
+    public function createToken(Argument $args): array
+    {
+        $arguments = $args->getArrayCopy();
+        $username = $arguments['username'];
+        $password = $arguments['password'];
```

Comment:
```suggestion
        $username = $arguments['username'] ?? null;
        $password = $arguments['password'] ?? null;
```
or check it in `isset` below instead.

Reply:
Please see line 36.

Reply:
You're checking that **those variables** are set and not null.

In line 36 you are assigning values from an array, but that array does not necessarily contain those keys.

Reply:
I misunderstood your comment, fixed via https://github.com/ibexa/graphql/compare/f43c65ec3a65bd6bce0f566bbf3faa34586ca37d..52e0d1ff5a80d75f62692edfb7a57069ee148f3c.